src/editors/Editor.test.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Editor, mapDispatchToProps } from './Editor';
+import { thunkActions } from './data/redux';
+import * as hooks from './hooks';
+
+const prepareEditorRefSpy = jest.spyOn(hooks, 'prepareEditorRef');
+
+const props = {
+  courseId: 'course-v1:edX+DemoX+Demo_Course',
+  blockType: 'html',
+  blockId: 'block-v1:edX+DemoX+Demo_Course+type@html+block@030e35c4756a4ddc8d40b95fbbfff4d4',
+  studioEndpointUrl: 'fakeurl.com',
+  initialize: jest.fn(),
+};
+
+jest.mock('./data/redux', () => ({
```

Comment:
you should not be mocking these out, but rather mocking out the hooks that trigger them.  This module has no need to mock any redux behavior.